<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="detections_sum_weight"                            default="0.4"/>
    <arg name="average_confidence_weight"                        default="0.5"/>
    <arg name="average_slope_weight"                             default="0.3"/>
    <arg name="average_density"                                  default="0.5"/>
    <arg name="comparison_resolution"                            default="100"/>
    <arg name="multisample_resolution"                           default="2"/>

    <node name="multicriteria_safety_distance" pkg="multicriteria_safety_distance" type="main_node.py" output="screen" respawn="true">
        <param name="detections_sum_weight"                      value="$(arg detections_sum_weight)"/>
        <param name="average_confidence_weight"                  value="$(arg average_confidence_weight)"/>
        <param name="average_slope_weight"                       value="$(arg average_slope_weight)"/>
        <param name="average_density"                            value="$(arg average_density)"/>
        <param name="comparison_resolution"                      value="$(arg comparison_resolution)"/>
        <param name="multisample_resolution"                     value="$(arg multisample_resolution)"/>
    </node>

</launch>
